src/device-registry/utils/create-event.js

```diff
@@ -1632,6 +1632,28 @@ const createEvent = {
       };
     }
   },
+  insertMeasurements: async(measurements)=>{
+    try {
```

Comment:
this looks good. Thanks @BenjaminSsempala !

Is there a possibility of reusing the already existing function of insert below, within this same file?

Otherwise, great works! I will attempt to test out the implementation later on today.